tests/TestCase/PluginTest.php

```diff
@@ -70,12 +72,28 @@ public function testSetDeprecationHandler()
         $first = $data['plugins']['DebugKit'][0];
         $this->assertEquals($first['message'], 'going away');
         $this->assertEquals($first['file'], __FILE__);
-        $this->assertEquals($first['line'], 48);
+        $this->assertEquals($first['line'], 50);
 
         $this->assertArrayHasKey('other', $data);
         $parsed = $data['other'][0];
         $this->assertEquals($parsed['message'], $message);
         $this->assertEquals($parsed['file'], 'src/Plugin.php');
         $this->assertEquals($parsed['line'], 51);
     }
+
+    /**
+     * Make sure the DebugkitMiddleware is not loaded while running PHPUnit
+     *
+     * @return void
+     */
+    public function test_middleware_not_loaded_in_tests()
```

Comment:
This should follow the standard naming convention.

Reply:
Sorry, still used to snake style from my work 🙈